app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, t("reset_pass") %>
+<h1><%= t("reset_pass") %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: password_reset_path(params[:id]) do |f| %>
+      <%= render "shared/error_messages", user: @user %>
+
+      <%= hidden_field_tag :email, @user.email %>
```

Comment:
khi dùng form_for, object f tạo ra có đầy đủ attribute của user nên khi ở dưới sử dụng các hàm f.label và f.text_field ta sẽ nhận được code html tương thích với attribute được gọi.
VD: dùng hàm này
<%= f.label :name %>
<%= f.text_field :name %>
ta sẽ nhận được code HTML như sau:
<label for="user_name">Name</label>
<input id="user_name" name="user[name]" type="text" />

còn nếu dùng form_tag thì sẽ không có attribute sẵn theo model như trên để tạo code html tương ứng